.env

```diff
@@ -0,0 +1,8 @@
+BINTRAY_KEY=f4g8ef@***223
```

Comment:
This is only temporary for testing, right?

Reply:
yep